src/Accelerators/NNPA/Conversion/ONNXToZHigh/ONNXLegalityCheck.cpp

```diff
@@ -266,6 +269,11 @@ bool meetPoolParamRestrictions(int64_t inputShape, int64_t kernelShape,
   return true;
 }
 
+void emitLegalityCheckMessage(Operation *op, StringRef message) {
+  LLVM_DEBUG(llvm::outs() << "[NNPA Legality Check] Warning: " << op->getLoc()
+                          << " runs on CPU. Reason: " << message << "\n");
+}
```

Comment:
Could you print out the operation name also? Sometime the loc is just a number and it's not easy to know which operation is.

Reply:
Added operation name and node name. Thanks!